imagersite/imager_images/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+from .views import photo_view,\
+        photo_gallery_view, album_view, album_gallery_view, library_view
+
+urlpatterns = [
+    path('photos/<photo_id>', photo_view, name='photo'),
```

Comment:
You should mark the data type of this url parameter as an `int` to ensure that the value passed back is at least a  number for the id.

Reply:
done.